File: VB/PreviewAndPrint/Form1.vb
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms

Namespace PreviewAndPrint

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            ' set advanced print options, if needed
            chartControl1.OptionsPrint.SizeMode = DevExpress.XtraCharts.Printing.PrintSizeMode.Zoom
            printableComponentLink1.Landscape = True
            printableComponentLink1.PaperKind = System.Drawing.Printing.PaperKind.A4
        End Sub

        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs)
            printableComponentLink1.ShowPreview()
        End Sub

        Private Sub simpleButton2_Click(ByVal sender As Object, ByVal e As EventArgs)
            printableComponentLink1.Print(String.Empty)
        End Sub
    End Class
End Namespace

File: CS/PreviewAndPrint/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace PreviewAndPrint {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // set advanced print options, if needed
            chartControl1.OptionsPrint.SizeMode = DevExpress.XtraCharts.Printing.PrintSizeMode.Zoom;
            printableComponentLink1.Landscape = true;
            printableComponentLink1.PaperKind = System.Drawing.Printing.PaperKind.A4;
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            printableComponentLink1.ShowPreview();
        }

        private void simpleButton2_Click(object sender, EventArgs e) {
            printableComponentLink1.Print(string.Empty);
        }
    }
}